imagebuildah/executor.go

```diff
@@ -531,7 +531,7 @@ func (b *Executor) Build(ctx context.Context, stages imagebuilder.Stages) (image
 	if err := cleanup(); err != nil {
 		return "", nil, err
 	}
-
+	logrus.Debugf("printing final image id %q", imageID)
```

Comment:
I've added this debug in my own self interest.  This is the 3rd time in the past few months that I've spent 10 or more minutes looking for where the final image id is printed in the code.  Hopefully going forward  I'll remember to run the bud command in debug and will be able to locate it more quickly.